patch-package
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
old mode 100644
new mode 100755
--- a/node_modules/react-spring/web.d.ts
+++ b/node_modules/react-spring/web.d.ts
@@ -10,3 +10,35 @@ declare const augmentedAnimated: typeof animated &
   }
 
 export { augmentedAnimated as animated }
+
+import { Controller } from './web.cjs';
+interface Controller {
+  new(props, config?): {
+    dependents: Set<any>;
+    isActive: boolean;
+    hasChanged: boolean;
+    props: any;
+    merged: any;
+    animations: any;
+    interpolations: any;
+    animatedProps: any;
+    configs: any[];
+    frame: any | undefined;
+    startTime: any | undefined;
+    lastTime: any | undefined;
+
+    update(props, ...start): any;
+
+    start(onEnd, onUpdate): Promise<any>;
+
+    stop(finished): void;
+
+    destroy(): void;
+
+    debouncedOnEnd(result): void;
+
+    getValues(): any;
+  };
+}
+const Ctrl = Controller as Controller;
+export { Ctrl as Controller }
\ No newline at end of file
old mode 100644
new mode 100755
old mode 100644
new mode 100755
